tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -165,14 +123,17 @@ var _ = ginkgov2.Describe("VM backup and restore tests", ginkgov2.Ordered, func(
 
 		url, err := getLatestCirrosImageURL()
 		gomega.Expect(err).To(gomega.BeNil())
-		err = v.EnsureDataVolumeFromUrl("openshift-cnv", "cirros-dv", url, "128Mi", 5*time.Minute)
+		err = v.EnsureDataVolumeFromUrl("openshift-virtualization-os-images", "cirros", url, "128Mi", 5*time.Minute)
+		gomega.Expect(err).To(gomega.BeNil())
+		err = v.CreateDataSourceFromPvc("openshift-virtualization-os-images", "cirros")
```

Comment:
this new way, CirrOS VM appears in cluster like a defualt one? (fedora, centos, etc)

Reply:
Yes, I figured out how to use the DataSource CRD, and now I get a "cirros" template alongside the others in the openshift-virtualization-os-images namespace. I was previously seeing permissions errors trying to clone the downloaded PVC, but having a real DataSource avoids that problem.

Reply:
niceeeee